CMP1903_A1_2324/Die.cs

```diff
@@ -13,11 +13,26 @@ internal class Die
          * and one method that rolls the die, returns and integer and takes no parameters.
          */
 
-        //Property
+        /// <summary> returns the currentValue of a randomly rolled 6 sided die. </summary>
 
+        // Private variable
+        private int _currentValue;
 
-        //Method
+        // Property
+        public int CurrentValue
+        {
+            get { return _currentValue; }
+            set { _currentValue = value; }
+        }
 
+        private static Random _random = new Random(); // Using static stops duplicate values
 
+        // Method
+        // Returns a random integer between 1 and 6
+        public int Roll()
+        {
```

Comment:
Some Function XML comments can be added to describe what is going on in the function alongside expected returns.